### DISCUSSION TITLE
启动 be 脚本报非法指令

### DISCUSSION THREAD

**Discussion by zzzzzzzs** [apache/doris#13094] [A - General / Q&A]
```shell
[root@zhaoshuo01 be]# sh bin/start_be.sh
bin/start_be.sh: 行 125: 25278 非法指令            (吐核)$LIMIT ${DORIS_HOME}/lib/doris_be "$@" 2>&1 < /dev/null
```

**Comment by dataalive** [comment]
https://doris.apache.org/docs/dev/install/source-install/compilation-with-ldb-toolchain?_highlight=avx2#compiling-doris

**Comment by JaneConan** [comment]
> ```shell
> [root@zhaoshuo01 be]# sh bin/start_be.sh
> bin/start_be.sh: 行 125: 25278 非法指令            (吐核)$LIMIT ${DORIS_HOME}/lib/doris_be "$@" 2>&1 < /dev/null
> ```

最后这个如何解决的。。。

**Reply by zzzzzzzs** [reply]
目前我用的docker编译的

**Reply by JaneConan** [reply]
我也是在 Docker 里启动的，跟您报错一样
